.envsample

```diff
@@ -0,0 +1,3 @@
+DEV_DATABASE_URL=postgres://shaluchandwani@127.0.0.1:5432/userschema
```

Comment:
You can use johndoe instead of your name.

Reply:
Changed the name as suggested.